packages/tenant-process/test/revokeDeclaredAttribute.test.ts

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable @typescript-eslint/no-floating-promises */
```

Comment:
[same](https://github.com/pagopa/interop-be-monorepo/pull/407#discussion_r1681251520).
Code smell